bsp/esp-box-3/idf_component.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
     public: true
 
   button:
-    version: "^2.5"
+    version: ">=2"
```

Comment:
Minimal version for >=IDF5 is 2.4 (there is new ADC driver)
- custom buttons are available from 2.5 (main button)